test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -242,13 +245,32 @@ class test_ucp_sockaddr : public ucp_test {
         UCS_TEST_MESSAGE << "server listening on " << m_test_addr.to_str();
     }
 
+     void start_listener(ucp_test_base::entity::listen_cb_type_t cb_type)
```

Comment:
need to use 4 symbols from the beginning of the line instead of 5 symbols as used here when implementing the function